src/core/include/openvino/op/util/framework_node.hpp

```diff
@@ -81,7 +81,7 @@ class OPENVINO_API FrameworkNode : public MultiSubGraphOp {
 
     explicit FrameworkNode(const OutputVector& inputs, size_t output_size = 1, size_t num_subgraphs = 0);
 
```

Comment:
Ant test or new model to enable by this implementation?

Reply:
This change should be tested by existing layer tests for types